fastlane/Fastfile

```diff
@@ -193,7 +193,7 @@ end
 
 def get_phc_version
   Dir.chdir(get_root_folder) do
-    return (sh "cat RevenueCatPurchasesCapacitor.podspec | grep \'\"PurchasesHybridCommon\", \' | awk \'{print($3)}\' | sed \"s/\'//g\"").strip
+    return (sh "cat RevenueCatPurchasesCapacitor.podspec | grep \"\'PurchasesHybridCommon\', \'\" | awk \'{print($3)}\' | sed \"s/\'//g\"").strip
```

Comment:
same here, I don't think `'` needs the `\`

```suggestion
    return (sh "cat RevenueCatPurchasesCapacitor.podspec | grep \"'PurchasesHybridCommon', '\" | awk '{print($3)}' | sed \"s/\'//g\"").strip
```

Reply:
Yup, just tested to confirm and it works fine 👍